package.json

```diff
@@ -80,6 +80,6 @@
     ]
   },
   "engines": {
-    "node": ">=10.16.0 <=12.x"
+    "node": ">=10.16.0 <12.x"
```

Comment:
Maybe a note on this. Did tun into this issue

Reply:
I checked https://nodejs.org/en/about/releases/, the active LTS release is `10.x`... I'll change `<` to `<=` @nerdlet